pyproject.toml

```
@@ -1,18 +1,17 @@
 [project]
 name = "ics_cobraOps"
```

Just FYI, package names should be specified with hyphens, not underscores. Same with lines `12-13`.

```suggestion
name = "ics-cobraOps"
```

---

Thanks. This was inherited. I'll fix it. (I thought these two are equivalent internally)